README.md

```diff
@@ -25,6 +25,7 @@ addon | version | summary
 [stock_available](stock_available/) | 9.0.1.1.0 | Stock available to promise
 [stock_available_immediately](stock_available_immediately/) | 9.0.1.1.0 | Ignore planned receptions in quantity available to promise
 [stock_available_mrp](stock_available_mrp/) | 9.0.1.0.0 | Consider the production potential is available to promise
+[stock_partner_lot](stock_partner_lot/) | 9.0.1.0.0 | Show and search for lots owned by partner
```

Comment:
Don't change this file. It's automatically changed by a bot when the module is merged. And squash the removal commit to avoid conflicts.